docs/src/functions/markdown-snippets/error-type-details.txt

```diff
@@ -0,0 +1 @@
+* For more information about the different types of errors that you may encounter when using IronCalc functions, visit our [Error Types](/features/error-types) page.
```

Comment:
I don't think this works. On my branch I was trying not to have a markdown file because they also get translated to html, but the problem is that including txt files does not work at all.

Reply:
My bad, just tested and this works perfectly well.